cdc/sink/producer/kafka/config.go

```diff
@@ -84,7 +84,7 @@ func (c *Config) CompleteByOpts(sinkURI *url.URL, replicaConfig *config.ReplicaC
 
 	s = params.Get("replication-factor")
 	if s != "" {
-		a, err := strconv.Atoi(s)
+		a, err := strconv.ParseInt(s, 10, 16)
```

Comment:
Why is this 16?

Reply:
because in L91 `	c.ReplicationFactor = int16(a)`